kafka-minion/values.yaml

```diff
@@ -119,3 +119,15 @@ serviceMonitor:
   # different matchLabels configuration, to make sure it matches this ServiceMonitor.
   # If this is defined, the release label of the service monitor will match kube-eagle's one
   # additionalLabels:
+
+# Create a service account by default
+# To use a service account not handled by the chart, set the name here
+# and set create to false
+serviceAccount:
+  create: false
+  name: default
+
+# Creates a PodSecurityPolicy and the role/rolebinding
+# allowing the serviceaccount to use it
+podSecurityPolicies:
```

Comment:
The variable name should be singular I think

Reply:
fixed